### DISCUSSION TITLE
Discord link is no longer valid

### DISCUSSION THREAD

**Discussion by 0xRapid** [valkey-io/valkey#1139] [General]
When opening an issue, I saw a Discord link; I tried to join the community and noticed it is no longer valid: https://discord.gg/zbcPa5umUB. Can someone please send me an updated invite? 

Besides that, I recommend you add the community link, such as Discord, to the website and the readme.

**Comment by melroy89** [comment]
There is mainly a [Matrix space](https://matrix.to/#/#valkey:matrix.org) and a Slack community afaik.
